src/LendingPool.sol

```diff
@@ -174,7 +174,10 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken {
             totalRealisedLiquidity += assets;
         }
 
-        _updateInterestRate();
+        uint64 _utilisation = uint64(realisedDebt / totalRealisedLiquidity);
```

Comment:
use a uint256 for variables stored in memory

Reply:
Now when calculating utility, we multiply realisedDebt by 10^5